js/models.js

```diff
@@ -1,13 +1,25 @@
+/* Todo model
+ * 
+ * cid {string} identity property given by BackboneJS used to track objects
+ * isComplete {bool} defaults to false, controls the class that shows a Todo as complete
+ * description {string} the text to show for the Todo 
+*/
+
 App.Models.Todo = Backbone.Model.extend({
     defaults: {
-        'isComplete': false,
+        isComplete: false,
     },
 
     toggleComplete: function () {
         this.set({ 'isComplete': !this.get('isComplete') });
         this.save();
     },
 
+    destroy: function(options = {}){
+        this.trigger('destroy', this, this.collection, options);
+        this.sync('delete', this, options);
+    },
```

Comment:
We talked about this outside this PR, but it's really important to actually call super here because this is missing core functionality like `model.stopListening();` which would cause memory leaks with dependent views.